### PR TITLE
Update container environment variables for bitnami/rabbitmq

### PR DESCRIPTION
**Pull request by msnwatson**
## What was the problem?

Followup to last PR

Associated tickets or Slack threads:
- #1007 

## How does this fix it?[^1]

Updates environment variables to be compatible with bitnami/rabbitmq

## How to test this PR
- Step 1
- Step 2[^secrel]

[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).

### CODE REVIEW

**Comment by msnwatson** [comment]
`container-healthchecks`, `api-gateway`, and `ee-ep-merge-end-to-end` will all fail until https://github.com/department-of-veterans-affairs/abd-vro/pull/2163 is merged

**Review by chengjie8 — APPROVED** [review]
LGTM

File: domain-ee/ee-ep-merge-app/src/requirements.txt [hunk 1]
```diff
@@ -1,5 +1,5 @@
 fastapi==0.103.*
-hoppy @ git+https://github.com/department-of-veterans-affairs/abd-vro.git@shared/lib-hoppy-0.2#subdirectory=shared/lib-hoppy
+hoppy @ git+https://github.com/department-of-veterans-affairs/abd-vro.git@shared/lib-hoppy-0.3#subdirectory=shared/lib-hoppy
```
Comment (dfitchett) [inline domain-ee/ee-ep-merge-app/src/requirements.txt:2]: Dont forget to tag the commit with `lib-hoppy-0.3`, and the version stored in the file `shared/lib-hoppy/hoppy/__init__.py` should be updated as well.

Comment (msnwatson) [reply domain-ee/ee-ep-merge-app/src/requirements.txt:2]: Yup, will do!

**Review by dfitchett — APPROVED** [review]

**Pull request merged**